### ISSUE TITLE
Tiff: IOError: cannot identify 16-bit TIF file

### ISSUE THREAD

**Issue by vigliensoni** [DDMAL/pil-rodan#1]
When trying to convert a TIF image to PNG with this wrapper, a "cannot identify image file" error is raised. This behavior seems to be related to the inability of PIL to handle images with individual channels > 8 bit (https://github.com/python-pillow/Pillow/issues/1888).

We should find a way of opening 16-bit TIF files.

**Comment by vigliensoni** [comment]
@deepio This seems to have been fixed in the latest Gamera release. Check https://github.com/hsnr-gamera/gamera/releases/tag/3.4.3

**Comment by vigliensoni** [comment]
@martha-thomae Could you please provide your insights with this issue?

**Comment by martha-thomae** [comment]
Hi @vigliensoni.
Yes, I also had some issues when trying to display TIF images in Rodan and when trying to convert them into PNG within Rodan and in my case these images were already 8-bit. **The issue** with my TIF images was that they used an uncommon **ColorSync Profile** (OS14K_A1_B8_oG_IT8_270109_editiert). When I changed this into a more standard profile like _Adobe RGB (1988)_ or _Generic RGB Profile_, the problem was solved. I can now view these images in Rodan and use them in different Rodan jobs such as _Pixel_ and any _PIL Conversion job_ (including conversion _to PNG_). All of these were impossible using the older (weirder) color profile.
